plugin-tsv/README.md

```diff
@@ -154,6 +154,23 @@ empty optional. If `"missingUsesDefaults"` is true, then, the values in
 The types are JSON-enhanced descriptors. See [types in the language
 description](../language.md#types) for details.
 
+It is also possible to use data from a remote server using a `.remotejsonconfig`:
+
+    {
+       "types": {
+          "foo": "s",
+          "bar": "qi",
+          "quux" "i"
+       },
+       "defaults": { "quux": 9000 },
+       "missingUsesDefaults": false,
+       "ttl": 10,
+       "url": "http://example.com/data"
```

Comment:
The data at that URL is expected to be a .jsonconfig file, right? If that's the case why do we need to specify the types twice?

Reply:
No, it's only the `"values"` component. The reason is that it gives you an upgrade path. If you are adding new attributes, you can safely have the server emit them and the Shesmu instances will ignore them until you update the configuration file.